themes/socialbase/templates/group/group--default.html.twig

```diff
@@ -42,7 +42,12 @@
 {% if content.field_group_description|render %}
   <div{{ attributes.addClass('card') }}>
     <div{{ content_attributes.addClass('card__body body-text') }}>
-      {{ content.field_group_description }}
+      {% if style == 'sky' %}
+        <div class="field--label">{% trans %} Introduction {% endtrans %}</div>
```

Comment:
We're introducing a label here. This field is labeled `description` in the backend. So labelling it `Introduction` in the front-end could be confusing to users (and also make a dis-joint from what you write to how it's interpreted).

We should evaluate (cc @xinyumaGG) whether this is indeed what we want. If we decide that this is the way forward then I feel we should also add this label to the default flavour. In the default flavour the group manager card is also already labelled and the description card is the only card that's not labelled.